controls/V-72225.rb

```diff
@@ -1,179 +1,210 @@
-
-    -This IS includes security measures (e.g., authentication and access controls)
-to protect USG interests--not for your personal benefit or privacy.
+    -Communications using, or data stored on, this IS are not private, are
+subject to routine monitoring, interception, and search, and may be disclosed
+or used for any USG-authorized purpose.
 
-    -Notwithstanding the above, using this IS does not constitute consent to PM, LE
-or CI investigative searching or monitoring of the content of privileged
-communications, or work product, related to personal representation or services by
-attorneys, psychotherapists, or clergy, and their assistants. Such communications
-and work product are private and confidential. See User Agreement for details.\"
+    -This IS includes security measures (e.g., authentication and access
+controls) to protect USG interests--not for your personal benefit or privacy.
 
-    Satisfies: SRG-OS-000023-GPOS-00006, SRG-OS-000024-GPOS-00007 ,
-SRG-OS-000228-GPOS-0008.
+    -Notwithstanding the above, using this IS does not constitute consent to
+PM, LE or CI investigative searching or monitoring of the content of privileged
+communications, or work product, related to personal representation or services
+by attorneys, psychotherapists, or clergy, and their assistants. Such
+communications and work product are private and confidential. See User
+Agreement for details.\"
   "
   impact 0.5
-  tag "severity": "medium"
   tag "gtitle": "SRG-OS-000023-GPOS-00006"
+  tag "satisfies": ["SRG-OS-000023-GPOS-00006", "SRG-OS-000024-GPOS-00007",
+"SRG-OS-000228-GPOS-00088"]
   tag "gid": "V-72225"
-  tag "rid": "SV-86849r2_rule"
+  tag "rid": "SV-86849r3_rule"
   tag "stig_id": "RHEL-07-040170"
-  tag "cci": "CCI-000048"
-  tag "nist": ["AC-8 a", "Rev_4"]
-  tag "cci": "CCI-000050"
-  tag "nist": ["AC-8 b", "Rev_4"]
-  tag "cci": "CCI-001384"
-  tag "nist": ["AC-8 c 1", "Rev_4"]
-  tag "cci": "CCI-001385"
-  tag "nist": ["AC-8 c 2", "Rev_4"]
-  tag "cci": "CCI-001386"
-  tag "nist": ["AC-8 c 2", "Rev_4"]
-  tag "cci": "CCI-001387"
-  tag "nist": ["AC-8 c 2", "Rev_4"]
-  tag "cci": "CCI-001388"
-  tag "nist": ["AC-8 c 3", "Rev_4"]
+  tag "cci": ["CCI-000048", "CCI-000050", "CCI-001384", "CCI-001385",
+"CCI-001386", "CCI-001387", "CCI-001388"]
+  tag "documentable": false
+  tag "nist": ["AC-8 a", "AC-8 b", "AC-8 c 1", "AC-8 c 2", "AC-8 c 2", "AC-8 c
+2", "AC-8 c 3", "Rev_4"]
   tag "subsystems": ["ssh"]
-  tag "check": "Verify any publicly accessible connection to the operating system
-displays the Standard Mandatory DoD Notice and Consent Banner before granting access
-to the system.
+  tag "check": "Verify any publicly accessible connection to the operating
+system displays the Standard Mandatory DoD Notice and Consent Banner before
+granting access to the system.
 
 Check for the location of the banner file being used with the following command:
 
 # grep -i banner /etc/ssh/sshd_config
 
-banner=/etc/issue
+banner /etc/issue
```

Comment:
I get that the STIG text says `/etc/issue` but technically, this should be `/etc/issue.net` since that is for network services and `/etc/issue` is for local logins and may differ. We may want to report this to DISA as a defect.